src/budget/Types.ts

```diff
@@ -82,7 +86,12 @@ export function validateBudgetState(data: unknown): BudgetState {
       'File format not supported. Please use an earlier version of BudgetBudget to open this file',
     );
   }
-
+  const semver = require('semver');
+  // Upgrade to 0.0.4 version format
+  if (semver.lt(version, '0.0.4')) {
+    (data as any).settings.ignorePendingTransactions = false;
+    (data as any).version = '0.0.4';
+  }
```

Comment:
Awesome! You thought about a migration!

Can we move `const semver = require('semver');` to the top and change to es module syntax `import semver from 'semver'`?